tools/cli/README.md

```diff
@@ -42,6 +43,15 @@ The CLI commands can be run from anywhere, and the changes will be made in which
 This command lets you build a project in the monorepo by selecting a project from a list, or providing one as an argument.
 
 - Build the Jetpack plugin: `jetpack build plugins/jetpack`
+- Build the Jetpack plugin for productioon: `jetpack build --production plugins/jetpack`
+
+**Build Extensions**: Build project extensions in the monorepo.
```

Comment:
Do we expect or need any other project to need this?

I'm okay just having `jetpack build-extensions` by itself running the jetpack product build-extensions.

Reply:
can we set the Jetpack plugin as the default project when there is not one is defined?

Reply:
A global command defined only for a particular case doesn't sound good to me since makes the API kind of inconsistent. Anyway, it's your call.

Reply:
addressed here 5d8a2e788b487291504b0bd5ff0dd4a59b5e53ea